lib/actions-oclif/device/init.ts

```diff
@@ -144,28 +144,40 @@ export default class DeviceInitCmd extends Command {
 
 	async downloadOsImage(path: string, deviceType: string, options: FlagsDef) {
 		const osVersion = options['os-version'] || 'default';
-		await runCommand(
-			`os download ${deviceType} --output '${path}' --version ${osVersion}`,
-		);
+		await runCommand([
+			'os',
+			'download',
+			deviceType,
+			'--output',
+			`'${path}'`,
```

Comment:
Does it not work with simply `path` instead of `'${path}'`? Perhaps after the change that I am suggesting in the other comment, so `capitanoRunAsync` takes an array of strings.

Reply:
I'm not sure, I was trying to ensure that nothing changed, but will test (would be good if we could avoid oclif/capitano differences).

Reply:
Works!